tests/test-components/src/main/java/org/mule/tests/api/LifecycleTracker.java

```diff
@@ -0,0 +1,12 @@
+package org.mule.tests.api;
+
+import org.mule.runtime.api.lifecycle.Disposable;
+import org.mule.runtime.api.lifecycle.Initialisable;
+import org.mule.runtime.core.api.context.MuleContextAware;
+
+import java.util.List;
+
+public interface LifecycleTracker extends Initialisable, Disposable, MuleContextAware {
```

Comment:
add javadocs

Reply:
Added for this interface and the base class. Do you think it's needed somewhere else?